packages/web/src/components/PageBanner.tsx

```diff
@@ -22,12 +25,14 @@ const PageBanner = ({ children, color, backButtonAction }: Props) => {
 
   return (
     <Box bgcolor={currentBannerColor || colors.black} mb={4} py={2}>
```

Comment:
```suggestion
    <Box bgcolor={currentBannerColor || colors.black} mb={4}>
```
Looks like we'll want to reduce this as well. Apologies, I know this is nitpicky 😓

Reply:
no worries, please no apologies needed. I will remove this `py={2}` as suggested and recommit.
😊 thank you.